manager.py

```diff
@@ -288,8 +286,17 @@ def pip_install_req(self, path_to_block):
         if not os.path.isfile(path_req):
             return True
 
+        try:
+
+            # Change directory to block
+            chdir(path_to_block)
+
+        except FileNotFoundError:
+            self.logger.error("Path '{0}' is invalid".format(path_to_block))
```

Comment:
Upon further review, this maintains the (albeit questionable) standard in the rest of this file, so it is fine as is, the functionality is equivalent.